browser/ftx/sources.gni

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//brave/components/ftx/browser/buildflags/buildflags.gni")
+
+brave_browser_ftx_sources = []
+brave_browser_ftx_deps = []
+
+if (ftx_enabled) {
+  brave_browser_ftx_sources += [
```

Comment:
is there a circular dependency here? If not it should just be added as a normal dep

Reply:
When I do it the normal way, I get the following, so I guess there is an unavoidable circular dependency because we need to use `"chrome/browser/external_protocol/external_protocol_handler.h"`

```
ERROR Dependency cycle:
  //chrome/browser:browser ->
  //brave/browser/ftx:ftx ->
  //chrome/browser:browser
```

Reply:
fixed via https://github.com/brave/brave-core/compare/6f4c503619e82a440ef7db9782288e922c37d5e4..f4d887c0baf39543331107f2e22f9b0ad4986c02